nemo/core/classes/common.py

```diff
@@ -257,13 +257,21 @@ def from_config_dict(cls, config: DictConfig):
 
         if ('cls' in config or 'target' in config) and 'params' in config:
             # regular hydra-based instantiation
+            if config.cls == 'nemo.collections.asr.modules.ConvASREncoder':
+                OmegaConf.set_struct(config, False)
+                config.quantize = quantize
+                OmegaConf.set_struct(config, True)
```

Comment:
Not needed to specialize code like this for a certain module, and config.cls is not guarenteed to exist since Hydra 1.x has `_target_` now. Better not override any such core method to inject parameters and handle it inside `ConvASREncoder` itself.